bagel/default.nix

```diff
@@ -96,6 +96,8 @@ in stdenv.mkDerivation {
 
   doCheck = true;
 
+  passthru.mpi = { inherit mpi; };
```

Comment:
That results in `passthru.mpi.mpi` but we want `passthru.mpi`. It should read `passthru.mpi = mpi` ( or  `passthru = {inherit mpi};` if you want to use the `inherit` )